CMakeLists.txt

```diff
@@ -86,45 +103,6 @@ if(NOT bgfx_FOUND)
   set(BGFX_CUSTOM_TARGETS
       OFF
       CACHE BOOL "Include convenience custom targets.")
-  set(BGFX_USE_DEBUG_SUFFIX
-      OFF
-      CACHE BOOL "Add 'd' suffix to debug output targets")
-
-  include(CMakeDependentOption)
-  option(OPENBLACK_BGFX_USE_SYSTEM_DEPS
-         "Search for system libraries for all 3rd parties of bgfx" OFF)
-  cmake_dependent_option(
-    OPENBLACK_BGFX_USE_SYSTEM_SPIRV_TOOLS
-    "Search for system spirv tools instead of building it" ON
-    "OPENBLACK_BGFX_USE_SYSTEM_DEPS" OFF)
-  cmake_dependent_option(
-    OPENBLACK_BGFX_USE_SYSTEM_SPIRV_CROSS
-    "Search for system spirv cross instead of building it" ON
-    "OPENBLACK_BGFX_USE_SYSTEM_DEPS" OFF)
-
-  if(OPENBLACK_BGFX_USE_SYSTEM_SPIRV_TOOLS)
-    find_library(SPIRVTOOLSOPT SPIRV-Tools-opt)
-    find_package(spirv-tools QUIET)
-    if(NOT spirv-tools_FOUND)
-      find_package(PkgConfig REQUIRED)
-      pkg_check_modules(SPIRV_TOOLS REQUIRED IMPORTED_TARGET GLOBAL SPIRV-Tools)
-      add_library(spirv-tools ALIAS PkgConfig::SPIRV_TOOLS)
-    endif()
-  endif()
-  if(OPENBLACK_BGFX_USE_SYSTEM_SPIRV_CROSS)
-    find_package(spirv_cross_core REQUIRED)
-    find_package(spirv_cross_glsl REQUIRED)
-    find_package(spirv_cross_msl REQUIRED)
-    find_package(spirv_cross_reflect REQUIRED)
-    add_library(spirv-cross INTERFACE)
-    target_link_libraries(
-      spirv-cross INTERFACE spirv-cross-core spirv-cross-glsl spirv-cross-msl
-                            spirv-cross-reflect)
-    if(WIN32)
-      find_package(spirv_cross_hlsl REQUIRED)
-      target_link_libraries(spirv-cross INTERFACE spirv-cross-hlsl)
-    endif()
-  endif()
```

Comment:
Please keep this, it cuts down build time when you have spirv installed system wide

Reply:
I'll move this to a custom BuildBgfx.cmake file - however I am slightly opposed to using this sort of thing at all as potentially your spirv-cross/tools could be different then the current version in bgfx's wonderful versionless repo

Reply:
fair, but I promise to keep the versions in sync ;)